src/main/java/com/thecookiezen/presentation/ClusterController.java

```diff
@@ -28,7 +29,7 @@
         return clusterRepository.getAll();
     }
 
-    @RequestMapping(value="clusters/form", method=RequestMethod.GET)
+    @RequestMapping(value = "clusters/form", method = RequestMethod.GET)
```

Comment:
\[Checkstyle\] WARNING: 'method def modifier' have incorrect indentation level 4, expected level should be 2\.